packages/editor/src/plugins/dropzone-image/index.ts

```diff
@@ -0,0 +1,70 @@
+import { EditorPluginType } from '@editor/types/editor-plugin-type'
+
+import { DropzoneImageEditor } from './components/editor/editor'
+import { BackgroundShape, DropzoneVisibility } from './types'
+import {
+  type EditorPlugin,
+  type EditorPluginProps,
+  object,
+  string,
+  child,
+  number,
+  list,
+  optional,
+} from '../../plugin'
+
+export const positionData = object({
+  top: number(0),
+  left: number(0),
+})
+
+export const layoutData = object({
+  width: number(200),
+  height: number(70),
+})
+
+export const answerData = object({
+  id: string(''),
+  image: child({ plugin: EditorPluginType.Image }),
+  text: child({ plugin: EditorPluginType.Text }),
+})
+
+export const answerZoneState = object({
+  id: string(''),
+  name: string('aaa'),
+  position: positionData,
+  layout: layoutData,
+  answers: list(answerData, 0),
+})
+
+const dropzoneImageState = object({
+  answerZones: list(answerZoneState, 0),
+  canvasShape: string(BackgroundShape.Unset),
+  canvasDimensions: object({
+    height: number(786),
+    width: number(786),
```

Comment:
This feedback might be to late, but like Kulla I'd prefer a relative / percentage based approach for positioning and sizing.
E.g. `786px` as the base width could change and is already different in some integration I think.
(same then goes for `layoutData`)

Reply:
Yes, we will explore this for the 2nd iteration, for now we are using the scaling solution, which works sufficiently for small screens

Reply:
The available space for the editor in the upcoming integrations will definitely vary. Sometimes it will be smaller than on serlo.org because the editor will be shown in an iframe on another page.
I guess this speaks for percentage positioning and sizing. But I am fine with this solution for now until we go live.